shared/actions/chat2/index.js

```diff
@@ -1599,9 +1599,10 @@ function* attachmentUpload(action: Chat2Gen.AttachmentUploadPayload) {
     if (ordinal) {
       yield Saga.put(Chat2Gen.createAttachmentUploaded({conversationIDKey, ordinal}))
     }
-  } catch (e) {}
-  // TODO better error
-  logger.warn('Upload Attachment Failed')
+  } catch (e) {
+    // TODO better error
+    logger.warn(`Upload Attachment Failed: ${e}`)
```

Comment:
debugging left in on purpose?

Reply:
An earlier PR moved the `logger.warn` out of the `catch`, so it was getting logged on every run.